### PR TITLE
Add oVirt ovirt_host_networks modules

### PR DESCRIPTION
**Pull request by machacekondra**
##### ISSUE TYPE

- New Module Pull Request
##### COMPONENT NAME

- ovirt_host_networks
##### SUMMARY

This PR adds new module to manage networks of the hosts in oVirt.

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks again to @machacekondra. This module is going into community review.

We notice that you are one of the maintainers (@machacekondra) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by machacekondra** [comment]
@mwperina

File: cloud/ovirt/ovirt_host_networks.py [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+module: ovirt_host_networks
+short_description: Module to manage host networks in oVirt
+version_added: "2.3"
+author: "Ondra Machacek (@machacekondra)"
+description:
+    - "Module to manage host networks in oVirt."
+options:
+    name:
+        description:
+            - "Name of the the host to manage networks."
```
Comment (mwperina) [inline cloud/ovirt/ovirt_host_networks.py]: I'd rephrase: 'Name of the the host to manage networks for.'

File: cloud/ovirt/ovirt_host_networks.py [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+            - "C(gateway) - Gateway in case of I(static) boot protocol is used."
+            - "C(version) - IP version. Either v4 or v6."
+    labels:
+        description:
+            - "List of names of the network label to be assigned to bond or interface."
+    check:
+        description:
+            - "If I(true) verify connectivity between host and engine."
+            - "If after changing the networks configuration the connectivity from host to the engine
+               is lost changes are rolled back."
```
Comment (mwperina) [inline cloud/ovirt/ovirt_host_networks.py]: I'd rephrase:

Network configuration changes will be rolled back if connectivity between engine and the host is lost after changing network configuration.

**Review by mwperina — CHANGES_REQUESTED** [review]

**Review by mwperina — APPROVED** [review]
Looks good to me, thanks! :+1:

**Comment by machacekondra** [comment]
shipit

**Comment by gregdek** [comment]
Thanks @machacekondra. Since you are one of the maintainers (@machacekondra) of this module and you commented with "shipit", we are marking this PR for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
Thanks again to @machacekondra. This module is going into community review.

We notice that you are one of the maintainers (@machacekondra) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]
